backend/src/services/appointment-service.ts

```diff
@@ -125,6 +150,7 @@ class AppointmentService extends GeneralService<IAppointmentJson, IAppointmentDt
         { where: { id } }
       );
       await t.commit();
+      return;
```

Comment:
The frontend expects a return value that can be parsed as JSON. Returning `void` as response will cause the frontend `util.http` method to crash.